### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by StrangeWeirdKitten**
There's three responses in the art of griefing.

This helps to solve the third.

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 1]
```diff
@@ -94,6 +94,7 @@
 
 /obj/machinery/chem_dispenser/Initialize()
 	. = ..()
+	message_admins("A reagent dispensor at [ADMIN_VERBOSEJMP(src)]")
```
Comment (ChungusGamer666) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm:97]: NICE FUCKING INITIALIZE SPAM
USE MAPLOAD.

Comment (BongaTheProto) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm:97]: > NICE FUCKING INITIALIZE SPAM USE MAPLOAD.

shut the fuck up and go back to coding bob

Comment (NullFag) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm:97]: He do be right tho

Comment (StrangeWeirdKitten) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: Are you happy?

**Pull request merged**